procedure/pull-request-merge.rst

```diff
@@ -266,8 +299,7 @@ working repo before that point.
    Edit the commit message to link it to *why* you are merging.  Say
    something like::
 
-     Merging branch/2023-01-06/speed-hax for GitHub pull request 93
-     <https://github.com/Ravenbrook/mps/pull/93>.
+     Merging branch/2023-01-06/speed-hax for GitHub pull request 93 <https://github.com/Ravenbrook/mps/pull/93>.
```

Comment:
m. This should probably say "#93" because GitHub recognizes that in commits and marks it up, even though the link is also recognized.

Reply:
Fix: Added '#' in d3a8ba556